bindings/kafka/kafka.go

```diff
@@ -171,3 +226,15 @@ func (k *Kafka) Read(handler func(*bindings.ReadResponse) error) error {
 func (consumer *consumer) Cleanup(sarama.ConsumerGroupSession) error {
 	return nil
 }
+func updateAuthInfo(config *sarama.Config, saslUsername, saslPassword string) {
+	config.Net.SASL.Enable = true
+	config.Net.SASL.User = saslUsername
+	config.Net.SASL.Password = saslPassword
+	config.Net.SASL.Mechanism = sarama.SASLTypePlaintext
+
+	config.Net.TLS.Enable = true
+	config.Net.TLS.Config = &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
The linter complains that `InsecureSkipVerify` is set to true. Is this needed?

Reply:
removed it